test/shared/utils.py

```diff
@@ -86,6 +90,8 @@ def expected_wheels(package_name, package_version):
             '{package_name}-{package_version}-cp36-cp36m-win_amd64.whl',
             '{package_name}-{package_version}-cp37-cp37m-win_amd64.whl',
             '{package_name}-{package_version}-cp38-cp38-win_amd64.whl',
+            '{package_name}-{package_version}-pp272-pypy_41-win32.whl',
+            '{package_name}-{package_version}-pp372-pp372-win32.whl',
```

Comment:
To be fixed in PyPy 7.2.1 or 7.3.0, whichever release is next?

Reply:
If by "fixed" you mean "changed", then indeed it will be different after the next release.

Reply:
OK, "made consistent/as I initially expected", then